algorand/test_contract.py

```diff
@@ -33,6 +33,9 @@
 
 import sys
 
+# sha256("portal_transfer(byte[])byte[]")[:4]
+portal_transfer_selector = Bytes("base16", "0x903f4535")   
```

Comment:
Can have something similar to the js-sdk one where you construct from Method from method signature and get selector